.storybook/preview-head.html

```diff
@@ -7,50 +7,21 @@
 
 <style>
   * {
-    --blue-500: #037dd6;
-    --blue-400: #1098fc;
     --gray-pre-bg: #f8f8f8;
-    --font-family-base: Euclid, Roboto, Helvetica, Arial, sans-serif;
     --font-family-monospace: Inconsolata, monospace;
     --font-size-code: 0.875rem;
-    --line-height-code: 19px;
-    --font-size-base: 1rem;
-    --line-height-base: 1.7;
-
-    font-family: var(--font-family-base) !important;
-  }
-
-  h1,
-  h2 {
-    font-weight: bold !important;
-  }
-
-  p,
-  li {
-    font-size: var(--font-size-base) !important;
-    line-height: var(--line-height-base) !important;
   }
 
   .docblock-source {
     background: var(--gray-pre-bg) !important;
   }
 
-  code {
+  .docblock-source code {
     font-family: var(--font-family-monospace) !important;
     font-size: var(--font-size-code) !important;
   }
-  code * {
+  .docblock-source code * {
     font-family: var(--font-family-monospace) !important;
     font-size: var(--font-size-code) !important;
   }
-
-  a {
-    color: var(--blue-500) !important;
-  }
-
-  a:hover,
-  a:active,
-  a:focus {
-    color: var(--blue-400) !important;
-  }
 </style>
```

Comment:
Removing styles that could possibly overwrite the UI in the storybook iframes and using storybook theme api instead.